src/main/java/edu/harvard/iq/dataverse/export/ddi/DdiExportUtil.java

```diff
@@ -1818,10 +1818,12 @@ private static void createVarDDI(XMLStreamWriter xmlw, JsonObject dvar, String f
 
         // labl
         if ((vm == null || !vm.containsKey("label"))) {
-            xmlw.writeStartElement("labl");
-            writeAttribute(xmlw, "level", "variable");
-            xmlw.writeCharacters(dvar.getString("label"));
-            xmlw.writeEndElement(); //labl
+            if(dvar.containsKey("label")) {
```

Comment:
Could you combine this if with the enclosing if?

Reply:
Decided to remove the redundant 'if' on 1825 below instead.

Reply:
If I'm not mistaken, you could still combine this `if`'s condition with the `(vm == null || !vm.containsKey("label"))` and have the same logic.

```
if ((vm == null || !vm.containsKey("label")) && dvar.containsKey("label")) {
```
(and remove the `}` on line 1826)

Reply:
if (vm == null || !vm.containsKey("label")) is true and dvar.containsKey("label") is false, the else would be triggered, causing vm.getString("label") to be called (line 1830) when it doesn't contain that key.

Reply:
Of course! 🤦